config/cms.php

```diff
@@ -381,4 +381,26 @@
 
     'restrictBaseDir' => true,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Back-end Service Worker
+    |--------------------------------------------------------------------------
+    |
+    | Allow plugins to run Service Workers in the back-end.
+    |
+    | WARNING: This should always be turned for security reasons as Service
+    | Workers can be hijacked and used to run XSS into the backend. Turning
+    | this feature on can create a conflict, if you have a front-end Service
+    | Worker running. The'scope' needs to be correctly set and not have a
+    | duplicate sub-folder structure on the front-end, otherwise it will run
+    | on both the front-end and back-end of your website.	
```

Comment:
no hyphens in frontend or backend please

Reply:
strange I already removed them.